packages/integrations/markdoc/src/index.ts

```diff
@@ -231,17 +237,6 @@ export const Content = createComponent({
 					},
 				});
 			},
-			'astro:server:setup': async ({ server }) => {
-				server.watcher.on('all', (event, entry) => {
-					if (prependForwardSlash(pathToFileURL(entry).pathname) === markdocConfigResultId) {
-						console.log(
-							yellow(
-								`${bold('[Markdoc]')} Restart the dev server for config changes to take effect.`
-							)
-						);
-					}
```

Comment:
I think what we can do is directly restart it here with `server.restart()`. It's also what `vite-plugin-restart` [does under the hood](https://github.com/antfu/vite-plugin-restart/blob/43461b5f362e31944502cefdc45ef9c1f3ebf9eb/src/index.ts#L90).

Before we had our custom container restart strategy, I wanted to make integrations call `server.restart()` too, but I was a bit too late. So if we could start here that would be great 😄

Reply:
Ah, I'll see if we can simplify to that instead!

Reply:
Update: done. Works great 🏆